presentation-compiler/src/main/scala/org/alephium/ralph/lsp/pc/workspace/build/dependency/downloader/BuiltInFunctionInfo.scala

```diff
@@ -0,0 +1,66 @@
+package org.alephium.ralph.lsp.pc.workspace.build.dependency.downloader
+
+import org.alephium.ralph.BuiltIn
+import org.alephium.ralph.BuiltIn.Category
+
+object BuiltInFunctionInfo {
+
+  /**
+   * This code is borrowed from `alephium-tools` project.
+   *
+   * @see <a href="https://github.com/alephium/alephium/blob/master/tools/src/main/scala/org/alephium/tools/BuiltInFunctions.scala">BuiltInFunctions.scala</a>
```

Comment:
would it make sense to have `tools` available as library? I could change that to publish it form the node

Reply:
I'm not sure. `tools` depends on `app` while pulls everything, I'm not sure if adding it as a dependency would increase the size of the resulting `ralph-lsp.jar`, would it?

Reply:
ah yes you right, that's start to be a bit heavy just for a case class